### ISSUE TITLE
Feature Request: Overlay Icons in Solution Explorer

### ISSUE THREAD

**Issue by gigios** [sboulema/TSVN#50]
I'm using this extension for the first time.
There is any possibility to shown an overlay icon on each file/folder shown in the solution explorer?
In this way it's very easy to identify the changed/commit/unlocked files.

**Comment by sboulema** [comment]
I hope you will like using this extension!

Please read this wiki about the overlay icons: [Source Control Integration](https://github.com/sboulema/TSVN/wiki/Source-Control-Integration)

**Comment by gigios** [comment]
Ok, thanks for the answer.
I'm using ankhsvn in the Visual Studio 2017 but unfortunately it's not available for Visual Studio 2019

**Comment by sboulema** [comment]
I edited the vsixmanifest of AnkhSVN to include support for VS2109, might work, might not, give it a try.

https://drive.google.com/open?id=1Prc1GTwdfZpEkD0GDKFI1Zi4BPevQ0JE

And I'll see if I can include the overlay icons in the future...

**Comment by gigios** [comment]
Unfortunately the installation does not works.
I receive an exception (translated from Italian):
This extension cannot be installed because the following references are missing:
Microsoft.VisualStudio.Component.CoreEditor

-Microsoft.VisualStudio.Component.CoreEditor (Microsoft.VisualStudio.Component.CoreEditor)
   in Microsoft.VisualStudio.ExtensionManager.EngineUtilities.EnsureNoMissingReferences(IEnumerable`1 missingRefs)
   in Microsoft.VisualStudio.ExtensionManager.PackageInstaller.PrepareEngineInstall(IDependencyGraph dGraph, IDependencyComparisonSeed seed, Component vsCompPkg, Boolean isProductComponent, Int32& totalInstallationSteps)
   in Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.PerformSetupEngineInstall(InstallableExtensionImpl extension, Boolean installPerMachine, Boolean isPackComponent, IDictionary`2 extensionsInstalledSoFar, List`1 extensionsUninstalledSoFar, IInstalledExtensionList modifiedInstalledExtensionsList, IProgress`1 progress, InstallFlags installFlags, AsyncOperation asyncOp, Version targetedVsVersion, IInstalledExtension& newExtension)
   in Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.InstallInternal(InstallableExtensionImpl extension, InstallFlags installFlags, IDictionary`2 extensionsInstalledSoFar, List`1 extensionsUninstalledSoFar, IInstalledExtensionList modifiedInstalledExtensionsList, AsyncOperation asyncOp, IProgress`1 progress, Version targetedVsVersion)
   in Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.BeginInstall(IInstallableExtension installableExtension, InstallFlags installFlags, AsyncOperation asyncOp, Version targetedVsVersion)
   in Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.InstallWorker(IInstallableExtension extension, InstallFlags installFlags, AsyncOperation asyncOp)

**Comment by gigios** [comment]
I have found how fix the missing reference error.
It's necessary to update not only the extension.vsixmanifest file BUT also the catalog.json in the vsix archive.
I have changed the prerequisites to:
`    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,)" DisplayName="Visual Studio core editor" />
`
without specify the latest version supported.
The same in the catalog.json file:
from:
`"Microsoft.VisualStudio.Component.CoreEditor":"[15.0,16.0)"}`
to
`"Microsoft.VisualStudio.Component.CoreEditor":"[15.0,)"}`
